Corona-Warn-App/build.gradle

```diff
@@ -222,6 +222,43 @@ task jacocoTestReportDeviceRelease(type: JacocoReport, dependsOn: 'testDeviceRel
     getExecutionData().from(fileTree(dir: "$buildDir", includes: ["jacoco/testDeviceReleaseUnitTest.exec"]))
 }
 
+task jacocoTestReportDeviceForTestersRelease(type: JacocoReport, dependsOn: 'testDeviceForTestersReleaseUnitTest') {
```

Comment:
Looks good.
In the future we could setup a task that iterates over all release build flavors and generates jacoco reports.
Did you evaluate that?
I would be okay with the duplication here for now. Just curious.